src/Logging/Logging.EventSource/test/EventSourceLoggerTest.cs

```diff
@@ -694,7 +694,7 @@ private static class EventTypes
 #if NETCOREAPP
             { "E5JS", (e) => VerifySingleEvent(e, "Logger2", EventTypes.MessageJson, 5, null, LogLevel.Critical,
                 @"""ArgumentsJson"":{""stringParam"":""bar"",""int1Param"":""23"",""int2Param"":""45""",
-                @$"""ExceptionJson"":{{""TypeName"":""System.Exception"",""Message"":""oops"",""HResult"":""-2146233088"",""VerboseMessage"":""System.Exception: oops{EscapedNewline()} ---\u003e System.Exception: inner oops") },
+                @$"""ExceptionJson"":{{""TypeName"":""System.Exception"",""Message"":""oops"",""HResult"":""-2146233088"",""VerboseMessage"":""System.Exception: oops{EscapedNewline()} ---\u003E System.Exception: inner oops") },
```

Comment:
Yep, this is correct. Since we are now using the `JavascriptEncoder`, it ends up using upper case hex digits. The previously built-in escaping logic within System.Text.Json was using lower case hex digits.